summernote/summernote
# Bugfix/#1674

#### What does this PR do?
- some buttons like bold, underline, italic, strike need their visual state to be refreshed immediately.
#### Where should the reviewer start?
- start with the bs3/module/Context.js with createInvokeHandlerAndUpdateState function
- it's called many times in the bs3/module/Button.js 
#### How should this be manually tested?
- before, when you clicked bold for example, button didn't have a pushed state or when you changed the family name, the dropdownlist gets updated only once you start to type in
- click on italic, bold, every other button concerned by createInvokeHandlerAndUpdateState() and see that visual state immediately changes. Family font name gets immediately updated.
#### Any background context you want to provide?
#### What are the relevant tickets?
#1674
#1990
#1738
#1673
#### Screenshots (if for frontend)
### Checklist
- checked that bold, underline, strike,  subscript, superscript, italic get immediately toggled.
- checked that font-family name gets immediately updated in the dropdown list.
- checked that clear button and similar buttons don't change visually.
- added all required tests OK in Button.spec.js in bs3/module
